src/components/BuildList.tsx

```diff
@@ -187,8 +187,21 @@ const BuildList: FunctionComponent = () => {
             }?`}</Typography>
           }
           onSubmit={() => {
+            let indexOfBuildDeleted = buildList.findIndex((e) => e.id === menuBuild.id);
+            let indexOfSelectedBuild = buildList.findIndex((e) => e.id === selectedBuild?.id);
             deleteBuild(buildDispatch, menuBuild.id)
               .then((b) => {
+                if (indexOfBuildDeleted === indexOfSelectedBuild) {
```

Comment:
On line 80 we have effect that does something similar
Could you try selecting build with null so then this effect gonna be executed?

Reply:
If we use that effect, the selection will jump to the first build and we will lose track of where we were in the build list if the user wants to delete a number of builds at the bottom. That is what I do normally - review and delete old builds near the bottom of the list. I thought it is easier to select the previous build thereby making it easier to visualize which build got deleted. Please let me know what you think and then I can proceed accordingly.